### DISCUSSION TITLE
serializing nugets doesnt work

### DISCUSSION THREAD

**Discussion by Doprez** [stride3d/stride#1622] [Q&A] [answered]
I am trying to keep usefull cross project scripts in a personal Nuget package. I am running into an issue where I cant access the component scripts unless I inherit in a new scrip that exists in the current stride project. I was looking over the 
 [consume packages](https://doc.stride3d.net/latest/en/manual/nuget/consume-packages.html) manual but I cant place Scripts in the Assets folder so the project can see them.

[my LOD component](https://github.com/Doprez/Doprez.Stride/blob/master/Doprez.Stride/ModelCategory/LOD.cs) reference script this is one example of something that cant be seen in projects that use the nuget package. [nuget repo](https://www.nuget.org/packages/Doprez.Stride)

Is there something I am not understanding or is this just something that currently doesnt work with gamestudio?

**Comment by manio143** [comment, marked as answer]
It's not expected for you to have to make the script abstract. And the whole thing is based around NuGet - the components in Stride packages are usually sealed and work fine.
Editor uses reflection to decide if a type is serializable, so I'm excluding any issues with post-compile generation of binary serializer.
It should work.
